README.md

```diff
@@ -3,3 +3,4 @@
 ## Attendees
 
 - Sushobhana Patra (3rd year, CSE)
+-Prabhat
```

Comment:
`- Prabhat` . Whitespace missing after hyphen. Read more about the markdown format.

Reply:
Ok Sir